im.nheko.Nheko.yaml

```diff
@@ -0,0 +1,247 @@
+      # https://gitlab.gnome.org/GNOME/libsecret/-/issues/49
+      - -Dgcrypt=false
+    sources:
+      - commit: 3fe635e64efd4b8dbc9ec3548b0bc8034c7665c4
+        tag: 0.20.4
+        type: git
+        url: https://gitlab.gnome.org/GNOME/libsecret.git
+  #- config-opts:
+  #    - -DCMAKE_BUILD_TYPE=Release
+  #    - -DAVIF_CODEC_AOM=ON
+  #    #- -DBUILD_SHARED_LIBS=OFF
+  #  buildsystem: cmake-ninja
+  #  name: libavif
+  #  sources:
+  #    - sha256: 66e82854ceb84a3e542bc140a343bc90e56c68f3ecb4fff63e636c136ed9a05e
+  #      type: archive
+  #      url: https://github.com/AOMediaCodec/libavif/archive/refs/tags/v0.10.1.tar.gz
+  #- config-opts:
+  #    - -DCMAKE_BUILD_TYPE=Release
+  #    - -DWITH_EXAMPLES=OFF
+  #    #- -DBUILD_SHARED_LIBS=OFF
+  #  buildsystem: cmake-ninja
+  #  name: libheif
+  #  sources:
+  #    - sha256: e1ac2abb354fdc8ccdca71363ebad7503ad731c84022cf460837f0839e171718
+  #      type: archive
+  #      url: https://github.com/strukturag/libheif/releases/download/v1.12.0/libheif-1.12.0.tar.gz
+  #- config-opts:
+  #    - -DCMAKE_BUILD_TYPE=Release
+  #    - -DKIMAGEFORMATS_HEIF=ON
+  #  buildsystem: cmake-ninja
+  #  name: KImageFormats
+  #  sources:
+  #    - commit: ae6b724824fc2fdf71d50dc7ae0052ad1551b25a
+  #      tag: v5.93.0
+  #      type: git
+  #      url: https://invent.kde.org/frameworks/kimageformats.git
+  - name: QtKeychain
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_TEST_APPLICATION=OFF
+      - -DQTKEYCHAIN_STATIC=ON
+      - -DBUILD_WITH_QT6=ON
+    buildsystem: cmake-ninja
+    sources:
+      - commit: 69f993c47efed7e557d79a30a367014d9a27d809
+        tag: 0.14.1
+        type: git
+        url: https://github.com/frankosterfeld/qtkeychain.git
+  - name: nlohmann
+    config-opts:
+      - -DJSON_BuildTests=OFF
+    buildsystem: cmake-ninja
+    sources:
+      - sha256: d69f9deb6a75e2580465c6c4c5111b89c4dc2fa94e3a85fcd2ffcd9a143d9273
+        type: archive
+        url: https://github.com/nlohmann/json/archive/v3.11.2.tar.gz
+  - name: kdsingleapplication
+    config-opts:
+      - -DKDSingleApplication_EXAMPLES=OFF
+      - -DKDSingleApplication_QT6=ON
+    buildsystem: cmake-ninja
+    sources:
+      - sha256: c92355dc10f3ebd39363458458fb5bdd9662e080cf77d91f0437763c4d936520
+        type: archive
+        url: https://github.com/KDAB/KDSingleApplication/releases/download/v1.0.0/kdsingleapplication-1.0.0.tar.gz
+  - name: re2
+    buildsystem: simple
+    build-commands:
+      - make static
+      - make prefix=/app static-install
+    sources:
+      - sha256: f89c61410a072e5cbcf8c27e3a778da7d6fd2f2b5b1445cd4f4508bee946ab0f
+        type: archive
+        url: https://github.com/google/re2/archive/refs/tags/2022-06-01.tar.gz
+  - name: gstreamer
+    buildsystem: meson
+    sources:
+      - commit: d2c02bb704b5804ca057fc7e6c7b16b4466fd7d5
+        tag: 1.22.12
+        type: git
+        url: https://gitlab.freedesktop.org/gstreamer/gstreamer.git
```

Comment:
Add `disable-submodules: true` here

Reply:
Thanks. I was looking for that option, but couldn't find it when I added that module originally!